app/overrides/spree/admin/products/_form/add_description_wysiwyg.html.haml.deface

```diff
@@ -0,0 +1,3 @@
+/ replace "[data-hook=admin_product_form_left] code[erb-loud]:contains('f.text_area :description')"
+%text-angular{'id' => 'product_description', 'ng-model' => 'product.description', 'name' => 'product[description]', 'class' => 'text-angular', 'ta-toolbar' => "[['bold','italics','clear']]"}
+  != @product[:description].to_s.html_safe
```

Comment:
Ah! That's not good... looking at it again I think the sanitization I added was sufficient, but I only added it to the serializer and forgot to add it to the model, which is what's being referenced here.